firefox-ios/Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -977,8 +1017,9 @@ class SearchViewController: SiteTableViewController,
             if !featureFlags.isFeatureEnabled(.firefoxSuggestFeature,
                                               checking: .buildAndUser) {
                 model.shouldShowSyncedTabsSuggestions = true
+                model.shouldShowPrivateModeFirefoxSuggestions = true
```

Comment:
curious - why do we set this to true when the feature flag is disabled?

Reply:
This is to ensure their accuracy, as when switching experiments, Firefox Suggest might leave them in a disabled state resulting in being hidden in the other experiment.